packages/neon/neon/lib/src/widgets/linear_progress_indicator.dart

```diff
@@ -3,28 +3,32 @@ part of '../../neon.dart';
 class NeonLinearProgressIndicator extends StatelessWidget {
   const NeonLinearProgressIndicator({
     this.visible = true,
+    this.alwaysThere = true,
```

Comment:
I don't think adding more confusing options to the interface is good.

What exactly is the usecase? That apps can force hide the appBar?
In this case I'd again suggest making a theming option through a ThemeExtension so one could disable this for the entire subtree.